File: app/src/main/java/xgen/mobiroo/com/mobirooapp/Country.java
package xgen.mobiroo.com.mobirooapp;

/**
 * Created by Raymond Zhuang on 2016-06-18.
 */
public class Country {
    private String m_short_name;
    private String m_full_name;

    public void setShortName(String name){
        m_short_name = name;
    }

    public void setfullName(String name){
        m_full_name = name;
    }

    public String getShortName(){
        return m_short_name;
    }

    public String getFullName(){
        return m_full_name;
    }
}

File: app/src/main/java/xgen/mobiroo/com/mobirooapp/Country.kt
package xgen.mobiroo.com.mobirooapp

/**
 * Created by Raymond Zhuang on 2016-06-18.
 */
class Country {
    lateinit var shortName: String
    lateinit var fullName: String
        private set

    fun setfullName(name: String) {
        fullName = name
    }
}
